src/TestingComponenets/Header.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Be sure to run code formatting on all of your JS/TS files... seeing some quote (single vs double) inconsistencies and extra line breaks in this and a few other files. Hopefully Prettier + Eslint can correct these things easily for you.